### DISCUSSION TITLE
Submission past deadline

### DISCUSSION THREAD

**Discussion by centarsirius** [community/community#17909] [GitHub Education]
I just came to know about github graduation 2022's deadline and unfortunately I had no idea about it previously. Since it's only been a week since the deadline has passed and the 7500 PR target hasn't been achieved yet, kindly allow me to participate. I have already submitted my PR to be merged. [MY PR](https://github.com/education/GitHubGraduation-2022/pull/8219#pullrequestreview-995825545)

**Comment by ettaboyle** [comment]
Flagging this for @juanpflores & @mlarasusan 👀
